apps/condo/domains/billing/schema/BillingReceipt.test.js

```diff
@@ -735,6 +735,105 @@ describe('BillingReceipt', () => {
                 }, '"data.services[0]"; Field "name" of required type "String!" was not provided.')
             })
         })
+        describe('Hooks', () => {
+            describe('receiver field', () => {
+                test('Should create recipient and set receiver field automatically', async () => {
+                    const [receipt, attrs] = await createTestBillingReceipt(admin, context, property, account)
+
+                    expect(attrs).not.toHaveProperty('receiver')
+                    expect(attrs).toHaveProperty(['recipient', 'tin'])
+                    expect(receipt).toHaveProperty(['recipient', 'tin'])
+                    expect(receipt).toHaveProperty(['receiver', 'tin'], receipt.recipient.tin)
```

Comment:
You don't need to check this twice
```suggestion
                    expect(receipt).toHaveProperty(['receiver', 'tin'], receipt.recipient.tin)
```